requirements_3.10.txt

```diff
@@ -0,0 +1 @@
+requirements.txt
```

Comment:
I don't think I fully understand what these files are for. We may want to consider if they can be replaced by an update to the "System Requirements" section of the README. If we leave them we should maybe add documentation about the purpose and what to do with them. Is there some implicit behavior going on with these that I am unaware of perhaps?

Reply:
I, too, don't understand the need for Python version-specific requirements.txt files. Also, this is likely rather non-standard.

Reply:
Ok, I'll ditch that, I'll hard code the python version we support.